easybuild/easyconfigs/a/alfa/alfa-1.1.1-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,49 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'PythonBundle'
+
+name = 'alfa'
```

Comment:
@branfosj official software name (which we try to stick to) seems to be `ALFA` rather than `alfa`?

Reply:
fixed